config/200-clusterrole.yaml

```diff
@@ -21,7 +21,7 @@ metadata:
     app.kubernetes.io/part-of: tekton-triggers
 rules:
   - apiGroups: [""]
-    resources: ["configmaps", "secrets", "services", "events"]
```

Comment:
Hmmm, so why does the EL reconciler need access to secrets in the first place 🤔 ? As far as I know, only the EL sink's SA should need access to secrets.

Reply:
Oooh ... interesting @dibyom 

I think you are right.  Ultimately it is the interceptors called within the sink that are getting the secret token for the git*/bitbucket/etc. verification that we have de-scoped to be local namespace only.

And I see no other use of the k8s secret client.

So that would imply not even creating the new 200-role.yaml and 201-rolebinding.yaml files I have currently in the PR, and instead updating the event listener / sink related yaml under the examples folder, right?

On the plus side, being this much of a newbie path makes me feel young :-) 

I'll work on some changes tomorrow / Friday and we'll see how the PR tests fair.

thanks